libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -51,9 +55,20 @@ typedef struct OVContext {
 typedef struct OVModel{
     OVContext ctx;
     DNNModel *model;
+#if HAVE_OPENVINO2
```

Comment:
looks that we support both ov2 and ov1?

Reply:
We support openvino. If it has openvino2 API, then we use openvino2 API.

Reply:
Will it work if there is only ov1 in the runtime but user builds FFmpeg against ov2 or inverse ?

Reply:
No it won't work. We need to make sure it uses same env when both compiling and running.